playground/src/pages/CodingEnvironment.tsx

```diff
@@ -221,15 +236,46 @@ export default function CodingEnvironment(): JSX.Element {
     });
   };
 
+  /**
+   * Shows the proof that is stored in the session.
+  */
+  const showProof = async () => {
+    if (!proof) {
+      setOutput("There is no proof to show. \nDid you prove the program?");
+      toast.error("Showing proof failed");
+      return;
+    }
+    toast.custom((t) => (
+      <div
+        className={`${
+          t.visible ? 'animate-enter' : 'animate-leave'
+        } max-w-full w-full bg-white shadow-lg rounded-lg flex ring-1 ring-black ring-opacity-5 overflow-hidden`}
+      >
+        <div className="flex-1 w-0 p-4 break-normal overflow-hidden">
+          <h1 className="text-lg font-medium text-gray-900"> This is your proof as Uint8Array </h1>
+          {proof.toString()}
```

Comment:
Seeing the proof like this isn't useful, here are some suggestions:

- change from toast to a overlay with a `<code>` block with a scrollbar and show the complete proof using hex-encoding
- hex-encode the proof, and show only its beginning

In both cases, it would be useful to know the proof size in bytes

Reply:
done, check it out